holoviews/plotting/util.py

```diff
@@ -309,6 +309,45 @@ def compute_sizes(sizes, size_fn, scaling_factor, scaling_method, base_size):
     return (base_size*scaling_factor*sizes)
 
 
+def get_axis_padding(padding):
+    """
+    Process a padding value supplied as a tuple or number and returns
+    padding values for x-, y- and z-axis.
+    """
+    if isinstance(padding, tuple):
+        if len(padding) == 2:
+            xpad, ypad = padding
+            zpad = 0
+        elif len(padding) == 3:
+            xpad, ypad, zpad = (padding, padding, padding)
```

Comment:
This is confusing to me...if the input is a 3-tuple, the return is that 3-tuple repeated 3 times in another 3-tuple?

Reply:
Hmm, yeah offhand this does look incorrect. Reminds me that I haven't actually written a single test for 3d plots yet, which probably would have uncovered an issue here.